migrations/2024_08_15_MakeVerifiedAtConsistent.pgsql

```diff
@@ -0,0 +1,28 @@
+-- For historical reasons, there are two ways to verify a user:
+-- verifiedAt IS NOT NULL (current timestamp is set when email is verified)
+-- verification IS NULL (random token is removed when email is verified)
+
+-- This migration populates the `verifiedAt` field for those 
+-- users that are only verified via verification column.
+
+
+-- MIGRATE PUPILS  --
+UPDATE pupil SET "verifiedAt" = "createdAt"
```

Comment:
I'm still not sure if this makes sense or if `updatedAt`/`CURRENT_DATE` makes more sense... or if it even matters. 🤷

Reply:
dont think it matters - but in case it does createdAt makes more sense - assuming createdAt is never null?

Reply:
I just checked, and it's never null :)